pkgs/data/sgml+xml/schemas/xml-dtd/xhtml1/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl, libxml2 }:
 
 stdenv.mkDerivation {
-  name = "xhtml1-20020801";
+  pname = "xhtml1";
+  version = "unstable-2002-08-01";
 
   src = fetchurl {
-    url = "http://www.w3.org/TR/xhtml1/xhtml1.tgz";
+    url = "https://www.w3.org/TR/xhtml1/xhtml1.tgz";
```

Comment:
Just in case we ever need it https://web.archive.org/web/20211212162042/https://www.w3.org/TR/xhtml1/xhtml1.tgz